e2e-tests/stream-mode-tests.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+#######################################
+# Function that prints messages
+# Arguments:
+#   anything that needs printing.
+#######################################
+function print_message() {
+  local print_prefix="E2E: STREAM MODE TEST:"
+  echo "${print_prefix} $*"
+}
+
+#######################################
+# Call common.sh to setup 
+# the testing env
+#  
+#######################################
+source ./e2e-tests/common.sh
+setup
+
+test_dir=$(mktemp -d -t analytics_tests__XXXXXX_streaming)
+
+command=(mvn compile exec:java -pl streaming-binlog \
+   -Dexec.args="--openmrsUserName=admin --openmrsPassword=Admin123 \
+   --openmrsServerUrl=http://localhost:8099/openmrs \
+   --openmrsfhirBaseEndpoint=/ws/fhir2/R4 \
+   --outputParquetPath='${test_dir}' --fhirSinkPath=http://localhost:8098/fhir --sinkUserName=hapi --sinkPassword=hai ")
```

Comment:
It seems the password has been changed, is this intentional? If the password is wrong, how did the streaming mode managed to push new resources to the HAPI sink?